Mogakco/Sources/Presentation/Login/ViewModel/AutoLoginViewModel.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  AutoLoginViewModel.swift
+//  Mogakco
+//
+//  Created by 신소민 on 2022/11/29.
+//  Copyright © 2022 Mogakco. All rights reserved.
+//
+
+import Foundation
+
+import RxCocoa
+import RxSwift
+
+final class AutoLoginViewModel: ViewModel {
+    
+    struct Input {
+        var viewWillAppear: Observable<Void>
+    }
+    
+    struct Output { }
+    
+    private let coordinator: AuthCoordinatorProtocol
+    private let autoLoginUseCase: AutoLoginUseCaseProtocol
+    var disposeBag = DisposeBag()
+    
+    init(
+        coordinator: AuthCoordinatorProtocol,
+        autoLoginUseCase: AutoLoginUseCaseProtocol
+    ) {
+        self.coordinator = coordinator
+        self.autoLoginUseCase = autoLoginUseCase
+    }
+    
+    func transform(input: Input) -> Output {
+        input.viewWillAppear
+            .withUnretained(self)
+            .flatMap { $0.0.autoLoginUseCase.load() }
+            .withUnretained(self)
+            .subscribe { viewModel, result in
```

Comment:
```suggestion
        input.viewWillAppear
            .withUnretained(self)
            .flatMap { $0.0.autoLoginUseCase.load() }
            .delay(.seconds(2), scheduler: MainScheduler.instance)
            .withUnretained(self)
            .subscribe { viewModel, result in
```
런치스크린이 보여지는 최소 시간이 보장되면 좋을 것 같아요! 1초 이내로 화면이 바뀌게되면 어색할 수 있을 것 같아서요
좋은 방법은 아니지만 쉽게 처리할 수 있는 방법으로는 delay를 주는 정도일 것 같습니다

Reply:
애니메이션을 보여주려면 딜레이가 있어야 하는데 너무 금방 넘어갔네요. 감사합니다.